judgments/models.py

```diff
@@ -30,7 +30,7 @@ def __init__(
         try:
             self.date = dateparser.parse(date)
         except ParserError:
-            self.date = date
+            self.date = None
```

Comment:
It might be worth flagging a rollbar error if the date is not empty and fails to parse; but it shouldn't raise.

Reply:
Added with tests